site/nci/envrun

```diff
@@ -19,4 +19,7 @@ export FFLAGS="-I /apps/openmpi/4.1.5/include/Intel ${FFLAGS:-}"
 # Pass through LD_LIBRARY_PATH
 export SINGULARITYENV_LD_LIBRARY_PATH="${LD_LIBRARY_PATH:-}"
 
+# Required for oneapi sanitizers to show addresses
+export SINGULARITYENV_APPEND_PATH="/apps/intel-oneapi/compiler/2023.2.0/linux/bin-llvm"
```

Comment:
I checked, this is indeed the latest version but it is strange that this is 2023.2.0 ?? Is there some package missing?

Reply:
Looks like the recent version of ifx uses a different directory structure on gadi which is why I didn't find it